File: week-07/day-2/Workshop 16.5.2023/url_fixer.py
# Accidentally I got the wrong URL for a funny subreddit. It's probably "odds" and not "bots"
# Also, the URL is missing a crucial component, find out what it is and insert it too!

url = "https//www.reddit.com/r/nevertellmethebots"
colon = ':'
add = url[:5] + colon + url[5:]
new_url = add.replace('bots', 'odds')
url = new_url
print(url)

File: week-02/day-4/Workshop 13.4.2023/src/UrlFixer.java
public class UrlFixer {
    public static void main(String[] args) {
        String url = "https//www.reddit.com/r/nevertellmethebots";

        // Accidentally I got the wrong URL for a funny subreddit. It's probably "odds" and not "bots"
        // Also, the URL is missing a crucial component, find out what it is and insert it too!
        // Try to solve it in more than one way using different String functions!
        System.out.println(url.indexOf("b"));
        System.out.println(url.length());
        buildAString(url);


    }
    public static void buildAString (String input) {
        StringBuilder sb = new StringBuilder(input);
        sb.insert(5, ":");
        sb.replace(38, 43, "odds");
        System.out.println(sb.toString());
    }
}
